orga/app/routes/authenticated/team.js

```diff
@@ -0,0 +1,18 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default Route.extend({
+
+  currentUser: service(),
+
+  beforeModel() {
+    this._super(...arguments);
```

Comment:
Mmmh faudrait qu'on s'accord sur l'utilisation de `this._super(...arguments);` est-ce qu'on le met partout ou seulement là où il est utile ? (question ouverte)

Reply:
alors, sachant qu'on sait jamais ce qu'il contient, que ça peut évoluer, qu'on a des bugs quand on ne le fait pas, pour moi on a plutôt intérêt à le mettre dans tous les cas.